projects/go-lib/src/lib/components/go-action-sheet/go-panel/go-panel.component.scss

```diff
@@ -116,4 +116,8 @@ button {
   &:focus {
     outline: none;
   }
+
+  &:disabled {
```

Comment:
This will only apply to the `button` element and not the `a`

Reply:
Thanks for pointing this out. Made some change,